modules/social_features/social_private_message/src/Plugin/UserExportPlugin/UserAnalyticsPrivateMessage.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\social_private_message\Plugin\UserExportPlugin;
+
+use Drupal\social_user_export\Plugin\UserExportPluginBase;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\user\UserInterface;
+
+/**
+ * Provides a 'UserAnalyticsPrivateMessage' user export row.
+ *
+ * @UserExportPlugin(
+ *  id = "user_private_message",
+ *  label = @Translation("Number of Private messages"),
+ *  weight = -190,
+ * )
+ */
+class UserAnalyticsPrivateMessage extends UserExportPluginBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * Returns the header.
+   *
+   * @return \Drupal\Core\StringTranslation\TranslatableMarkup|string
+   *   The header.
+   */
+  public function getHeader() {
```

Comment:
For all `getHeader()` and `getValue()` functions we can simply use:
```/**
 * {@inheritdoc}
 */```

Reply:
you're right. I'll change it